calcom/cal.com
# fix: Attach `credentialId` to location option

## What does this PR do?



This PR adds a missing field `credentialId` to event type location options. This credentialId is expected in `handleNewBooking` under `conferenceCredentialId`. For team events, the default host is not always present but it was still falling back to their location option.

Fixes # (issue)



## Type of change





## How should this be tested?



- On the event type setup tab, the location options should contain the `credentialId`
- Install a video app that is scoped to a team
- Set that team installed location to a team event type
- When creating a new booking the `conferenceCredentialId` in `handleNewBooking` should be defined

## Mandatory Tasks


## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
